docs/src/integrations/retrying-transactions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Retrying Transactions
+---
+
+# Retrying Transactions
+
+On some occasions, a seemingly valid transaction may be dropped before it is included in a block. This most often occurs during periods of network congestion, when an RPC node fails to rebroadcast the transaction to the [leader](https://docs.solana.com/terminology#leader). To an end-user, it may appear as if their transaction disappears entirely. While RPC nodes are equipped with a generic rebroadcasting algorithm, application developers are also capable of developing their own custom rebroadcasting logic.
+
+## Facts
+
+:::note
+Fact Sheet
+
+- RPC nodes will attempt to rebroadcast transactions using a generic algorithm
+- Application developers can implement their own custom rebroadcasting logic
+- Developers should take advantage of the `maxRetries` parameter on the `sendTransaction` JSON-RPC method
+- Developers should enable preflight checks to raise errors before transactions are submitted
+- Before re-signing any transaction, it is **very important** to ensure that the initial transaction’s blockhash has expired
+  :::
```

Comment:
```suggestion
:::
```

docusaurus doesn't like the leading whitespace.  it's currently rendering everything from `:::note` to `:::note` as a `Note` and not rendering the second note as a `Note`

Reply:
hmm think prettier did that. May cause issues for anyone that runs the formatting